barcodes_generator_abstract/README.rst

```diff
@@ -69,20 +69,20 @@ To configure this module, you need to:
 * Select a Nomenclature
 * Create or select a rule
 
-.. image:: https://raw.githubusercontent.com/OCA/stock-logistics-barcode/14.0/barcodes_generator_abstract/static/description/barcode_rule_tree.png
+.. image:: https://raw.githubusercontent.com/OCA/stock-logistics-barcode/15.0/barcodes_generator_abstract/static/description/barcode_rule_tree.png
 
 * For manual generation, set:
     * 'Base set Manually' in 'Generate Type'
     * Set the model
 
-.. image:: https://raw.githubusercontent.com/OCA/stock-logistics-barcode/14.0/barcodes_generator_abstract/static/description/barcode_rule_form_manual.png
+.. image:: https://raw.githubusercontent.com/barcodes_generator_abstract/static/description/barcode_rule_form_manual.png
```

Comment:
It seems you are still editing manually things, and there are a lot of incorrect links. I only point to the first, but you should consider seriously to use the automatic generator or let the bot to do it, but don't do it incorrectly provoking double diff.